build/Build.cs

```diff
@@ -478,15 +478,12 @@ void CompressCalamariProject(Project project)
         Target PackCalamariConsolidatedNugetPackage =>
             _ => _.DependsOn(PackageConsolidatedCalamariZip)
                   .Executes(() =>
-                            {
-                                var releaseNotes = IsLocalBuild ? "Local" : File.ReadAllText(RootDirectory / "releasenotes" / "ReleaseNotes.md");
-                                
-                                NuGetPack(s => s.SetTargetPath(BuildDirectory / "Calamari.Consolidated.nuspec")
-                                            .SetProperty("releaseNotes", releaseNotes)
-                                            .SetBasePath(BuildDirectory)
-                                            .SetVersion(NugetVersion.Value)
-                                            .SetOutputDirectory(ArtifactsDirectory));
-                            });
+                  {
+                      NuGetPack(s => s.SetTargetPath(BuildDirectory / "Calamari.Consolidated.nuspec")
+                          .SetBasePath(BuildDirectory)
+                          .SetVersion(NugetVersion.Value)
+                          .SetOutputDirectory(ArtifactsDirectory));
+                  });
```

Comment:
Need to back-port this so it builds properly on TC. See: https://github.com/OctopusDeploy/Calamari/pull/1058